3-2-version-control.qmd

```diff
@@ -33,6 +33,117 @@ Learning Git can be daunting 🙀. Historically it was developed by software eng
 That makes it particularly useful for us when writing code, but it can also feel a bit too nerdy in the beginning.
 I promise though, once you've got the hang of it, you'll love it!
 
+### Installing git on your computer 💻
+
+Depending on the system you are working on, there are different ways to install git.
+
+For the latest stable version for Linux Ubuntu run this command in your terminal:
+
+```bash
+apt-get install git
+```
+If you are on Mac you can use homebrew to install it in your terminal. 
+Install [homebrew](https://brew.sh/) if you don't already have it, then type:
+
+```bash
+brew install git
+```
+
+If you are working on Windows, I recommend installing Git Bash. This provides you with a terminal where you can work similarly to the terminal on Mac and Linux, using the same commands. Git Bash also automatically installs Git for you. You can download Git for Windows [here](https://gitforwindows.org/).
+
+You can visit [git-scm.com](https://git-scm.com/downloads) to get more information on how you can download git for your system.
+
+For R users, I recommend following [these](https://gitlab.com/HeidiSeibold/setup-git-rstudio-gitlab) instructions.
+
+### Create a new Repository on GitHub
```

Comment:
Should we use Installing/Creating/Cloning/... or Install/Create/Clone/...? Either is good, but I would stay consistent.